src/routes/docs/products/auth/security/+page.markdoc

```diff
@@ -39,4 +39,8 @@ Appwrite uses a [permissions model](/docs/advanced/platform/permissions) coupled
 With all Appwrite services, including databases and storage, access is granted at the collection, bucket, document, or file level.
 These permissions are enforced for client SDKs and server SDKs when using JWT, but are ignored when using a server SDK with an API key.
 
-{% partial file="auth-security.md" /%}
\ No newline at end of file
+{% partial file="auth-security.md" /%}
+
+# Session alerts {% #session-alerts %}
+
+Set up email alert notifications for your users so that whenever multiple sessions are created using their user account information, such as their email address, they will be alerted to the new session. The alert will be sent as an email, so user accounts must have a valid and existing email address to receive these notifications. However, users will not receive email notifications when creating their first session or using [Magic URL](/docs/products/auth/magic-url) login or [Email OTP](/docs/products/auth/email-otp) login. To toggle session alerts, navigate to **Auth** > **Security** > Session Alerts.
```

Comment:
> Set up email alert notifications

no need to say both alert and notifications.

> multiple sessions 

This makes it sound like multiple sessions created at the same time.

> multiple sessions are created using their user account information, such as their email address

Can this be made more concise?

> The alert will be sent as an email, so user accounts must have a valid and existing email address to receive these notifications.

Let's skip this for brevity. It should be obvious that email alerts won't go out 

> However, users will not receive email notifications

Good idea including caveats.

> using [Magic URL](/docs/products/auth/magic-url) login or [Email OTP](/docs/products/auth/email-otp) login

what makes you say this?

This 1 big chunk/paragraph is a bit big. Let's break it apart into separate paragraphs.